lib/types/cql-utils.js

```diff
@@ -7,19 +7,76 @@ const TimeUuid = require("./time-uuid");
 const Uuid = require("./uuid");
 const Duration = require("./duration");
 
+/**
+ * Ensures the value isn't one of may ways to express null value
+ * @param {string} message Message of error, in case value is "null"
```

Comment:
Why not ``Error message``? ``Message of error`` sounds like Yoda. ``if`` instead of ``in case``.